modules/TaskListHeaderPresenter.js

```diff
@@ -0,0 +1,37 @@
+import iconArrowsCcw from '../src/icon-arrows-ccw.svg';
+
+class TaskListHeaderPresenter {
+  #completedCount;
+
+  #listStats;
+
+  #view;
```

Comment:
Please check if these declarations are needed. They are also created in the constructor.

Reply:
I'll try to remove these, but they are required for proper function.